### ISSUE TITLE
hi, can you share the dataset you use for training? thanks

### ISSUE THREAD

**Issue by catalysts-higher** [hminle/car-behavioral-cloning-with-pytorch#3]
I tried to train your CarSimpleModel with data I collected from the simulator, but the loss never decrease, and the trained model can get through small curvature turns but struggled with sharp turns. But you model3.h5 works well. I can not figure out where is the problem. My dataset has 8037 frames. (3 images each frame). any suggestions? thank you very much!
